example/platform.html

```diff
@@ -97,8 +97,9 @@
 <script>
   function initViewer() {
     var viewerHost = document.getElementById('viewerHost');
-    var ampUrl = 'https://www.ampproject.org/';
-    var viewer = new AmpViewer(viewerHost, ampUrl);
+    var ampUrl = 'https://www-ampproject-org.cdn.ampproject.org/v/s/www.ampproject.org/?amp_js_v=0.1#origin=http://localhost:8000';
```

Comment:
I created a few methods inside viewer.js to do this, not sure if it's better to move them to helper files or not